spec/models/ems_refresh/parsers/kubernetes_spec.rb

```diff
@@ -22,12 +22,12 @@
                        :registry   => nil},
 
                       {:image_name => "host/subname/example",
-                       :image      => {:name => "subname/example", :tag => nil, :image_ref => example_ref},
-                       :registry   => {:name => "host", :host => "host", :port => nil}},
+                       :image      => {:name => "host/subname/example", :tag => nil, :image_ref => example_ref},
```

Comment:
Keeping `host` here is confusing because I'd expect it to appear in the registry (as `:name`). It should be renamed to something else.